torch_xla/csrc/aten_xla_type.cpp

```diff
@@ -1365,6 +1367,33 @@ at::Tensor XLANativeFunctions::expand(const at::Tensor& self,
       bridge::GetXlaTensor(self), torch::lazy::ToVector<int64_t>(size)));
 }
 
+at::Tensor XLANativeFunctions::expand(const at::Tensor& self,
+                                      c10::SymIntArrayRef size, bool implicit) {
+  XLA_FN_COUNTER("xla::");
+  std::vector<c10::SymInt> _sizes = torch::lazy::ToVector<c10::SymInt>(size);
+  std::vector<torch::lazy::NodePtr> size_nodes;
+  std::vector<int64_t> upper_bounds;
+  std::vector<bool> dynamic_dims;
+  /* TODO: move this code to a helper function */
```

Comment:
Considering to move this code to a helper function. My candidates are creating a new helper file, `helpers.h`, or `tensor_util.h`? @JackCaoG wdyt?